ia/src/protocole.py

```
@@ -0,0 +1,128 @@
+
+    while client["is_alive"]:
+        execute_command(client, "Broadcast", "Hello")
+        response = client["socket"].recv(1024).decode()
+
+        buffer += response
+
+        while "\n" in buffer:
+            message, buffer = buffer.split("\n", 1)
+
+            if message:
+                if (message == "dead\n"):
```

Pylint (3.3.7)

[convention] Unnecessary parens after 'if' keyword

(C0325)